wakuv2/nwaku.go

```diff
@@ -816,42 +830,44 @@ func (w *Waku) retryDnsDiscoveryWithBackoff(ctx context.Context, addr string, su
 	}
 }
 
+*/
+
 func (w *Waku) discoverAndConnectPeers() {
-	fnApply := func(d dnsdisc.DiscoveredNode, wg *sync.WaitGroup) {
-		defer wg.Done()
-		if len(d.PeerInfo.Addrs) != 0 {
-			go w.connect(d.PeerInfo, d.ENR, wps.DNSDiscovery)
-		}
-	}
+	var addrsToConnect []multiaddr.Multiaddr
+	nameserver := "1.1.1.1"
```

Comment:
What should I add here? For sure there's a better option than hardcoding this, but want to know how is it currently done :))

Reply:
I'd say for now just add `TODO-nwaku` ;P
Jokes apart (or not) I think the best would be to avoid passing the `nameserver` parameter and instead, have the complexity of picking up the appropriate dns-server within `libwaku`.